addons/web/static/src/model/relational_model/dynamic_list.js

```diff
@@ -329,7 +331,25 @@ export class DynamicList extends DataPoint {
             const resIds = unique(validSelection.map((r) => r.resId));
             const context = this.context;
             try {
-                await this.model.orm.write(this.resModel, resIds, changes, { context });
+                const standardChanges = {};
+                const operationsChanges = {};
+                Object.entries(changes).forEach(
+                    ([key, val]) =>
+                        ((val instanceof FieldOperator ? operationsChanges : standardChanges)[key] =
+                            val)
+                );
+
+                if (Object.keys(operationsChanges).length) {
```

Comment:
the two routes should be mutually exclusive
the custom route could handle the simple case as well if both standard and operations changes can be be done together

Reply:
or better yet, only at the list_controller level, around onWillSaveMulti